include/swift/Option/Options.td

```diff
@@ -368,6 +368,10 @@ def emit_objc_header_path : Separate<["-"], "emit-objc-header-path">,
          ArgumentIsPath]>,
   MetaVarName<"<path>">, HelpText<"Emit an Objective-C header file to <path>">;
 
+def static_library : Flag<["-"], "static">,
```

Comment:
Nitpick: the name of the option should match the name of the flag. (It'll get concatenated to `OPT_` so it shouldn't collide. Unless you already tried it and I'm wrong?)